src/Sylius/Behat/Context/Ui/Admin/ManagingShipmentsContext.php

```diff
@@ -0,0 +1,52 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Sylius\Behat\Context\Ui\Admin;
+
+use Behat\Behat\Context\Context;
+use Sylius\Behat\Page\Admin\Crud\IndexPageInterface;
+use Webmozart\Assert\Assert;
+
+final class ManagingShipmentsContext implements Context
+{
+    /** @var IndexPageInterface */
+    private $indexPage;
+
+    public function __construct(IndexPageInterface $indexPage)
+    {
+        $this->indexPage = $indexPage;
+    }
+
+    /**
+     * @When I browse shipments
+     */
+    public function BrowseShipments(): void
+    {
+        $this->indexPage->open();
+    }
+
+    /**
+     * @Then I should see single shipment in the list
+     */
+    public function iShouldSeeSingleShipmentInList(): void
+    {
+        Assert::eq(1, $this->indexPage->countItems());
+    }
+
+    /**
+     * @Then the shipment of the :orderNumber order should be :shippingState
+     */
+    public function ShipmentOfOrderShouldBe(string $orderNumber, string $shippingState): void
+    {
+        Assert::true($this->indexPage->isSingleResourceOnPage(['order_number' => $orderNumber, 'state' =>$shippingState]));
+    }
+
+    /**
+     * @Then I should see two shipments in the list
+     */
+    public function ShouldSeeTwoShipmentsInList(): void
+    {
+        Assert::eq(2, $this->indexPage->countItems());
```

Comment:
If we put a `:count` parameter instead of `two` in the step, we could pass it as a function argument and make this step more universal